cmd/manager/main.go

```diff
@@ -17,157 +17,98 @@ limitations under the License.
 package main
 
 import (
-	"context"
 	"flag"
 	"os"
-	"runtime"
-	"time"
-
-	"github.com/openebs/node-disk-manager/pkg/apis"
-	"github.com/openebs/node-disk-manager/pkg/controller"
-	"github.com/openebs/node-disk-manager/pkg/env"
-	ndmlogger "github.com/openebs/node-disk-manager/pkg/logs"
-	"github.com/openebs/node-disk-manager/pkg/setup"
-	"github.com/openebs/node-disk-manager/pkg/upgrade"
-	"github.com/openebs/node-disk-manager/pkg/upgrade/v040_041"
-	"github.com/openebs/node-disk-manager/pkg/upgrade/v041_042"
-	"github.com/openebs/node-disk-manager/pkg/version"
-
-	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
-	"github.com/operator-framework/operator-sdk/pkg/leader"
-	"github.com/operator-framework/operator-sdk/pkg/ready"
-	sdkVersion "github.com/operator-framework/operator-sdk/version"
-	"k8s.io/klog"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	openebsiov1alpha1 "github.com/openebs/node-disk-manager/api/v1alpha1"
+	blockdevice "github.com/openebs/node-disk-manager/controllers/blockdevice"
+	blockdeviceclaim "github.com/openebs/node-disk-manager/controllers/blockdeviceclaim"
+	//+kubebuilder:scaffold:imports
+)
+
+var (
+	scheme   = runtime.NewScheme()
+	setupLog = ctrl.Log.WithName("setup")
 )
 
-//ReconciliationInterval defines the triggering interval for reconciliation operation
-const ReconciliationInterval = 5 * time.Second
+func init() {
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
-func printVersion() {
-	klog.Infof("Go Version: %s", runtime.Version())
-	klog.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
-	klog.Infof("operator-sdk Version: %v", sdkVersion.Version)
-	klog.Infof("Version Tag: %s", version.GetVersion())
-	klog.Infof("Git Commit: %s", version.GetGitCommit())
```

Comment:
Need to include this information while starting the operator so that we can know which version is being run.